zulipterminal/model.py

```diff
@@ -126,7 +126,11 @@ def __init__(self, controller: Any) -> None:
 
         self.unread_counts = classify_unread_counts(self)
 
-        self.active_emoji_data = unicode_emojis.EMOJI_DATA
+        custom_emojis = self.fetch_custom_emojis()
+        all_emojis = (list(unicode_emojis.EMOJI_DATA.items())
+                      + list(custom_emojis.items()))
+        self.active_emoji_data = OrderedDict(sorted(all_emojis,
+                                                    key=lambda e: e[0]))
```

Comment:
If this handles the 'custom emoji can replace unicode emoji' case, then can we add a test case for it?

Reply:
Added.